.github/workflows/main.yml

```diff
@@ -40,21 +40,27 @@ jobs:
       - run: yarn install --frozen-lockfile
 
       - name: 'Basic Tests'
-        uses: GabrielBB/xvfb-action@v1
-        with:
-          run: yarn test-fast
+        run: yarn test-fast
 
-      # runs slow end-to-end tests, parallelizing yarn and npm cases
-      - name: 'End-to-end Tests: Yarn'
-        uses: GabrielBB/xvfb-action@v1
+  # runs slow end-to-end tests, parallelizing yarn and npm cases
+  test-end-to-end:
+    name: End-to-end tests (${{ matrix.os }})
```

Comment:
@bendemboski is it possible to add npm/yarn here, so we can see from the title of the run if it was npm or yarn, without needing to go into the action details?